pkg/namespace/namespace.go

```diff
@@ -0,0 +1,43 @@
+package namespace
+
+import (
+	"fmt"
+	"strconv"
+
+	corev1Listers "k8s.io/client-go/listers/core/v1"
+)
+
+type NamespaceInfo struct {
+	lister              corev1Listers.NamespaceLister
+	rateLimitAnnotation string
+}
+
+func NewNamespaceInfo(lister corev1Listers.NamespaceLister, rateLimitAnnotation string) *NamespaceInfo {
+	return &NamespaceInfo{
+		lister:              lister,
+		rateLimitAnnotation: rateLimitAnnotation,
```

Comment:
Update the README with how to set the annotation(s) and the expected value format(s).

Reply:
added to readme